packages/jsii-sampiler/test/python/expressions.test.ts

```diff
@@ -0,0 +1,61 @@
+import { expectPython } from "./python";
+
+test('as-expression', () => {
+  expectPython(`
+  console.log(3 as number);
+  `, `
+  print(3)
+  `);
+});
+
+test('prefix unary expression', () => {
+  expectPython(`
+  console.log(-3);
+  `, `
+  print(-3)
+  `);
+});
+
+test('ellipsis at a random place', () => {
+  expectPython(`
+  callThisFunction(foo, ...);
+  `, `
+  call_this_function(foo, ...)
+  `);
+});
+
+test('string interpolation', () => {
+  expectPython([
+  'const x = "world";',
+  'const y = "well";',
+  'console.log(`Hello, ${x}, it works ${y}!`);',
+  ].join('\n'), `
+  x = "world"
+  y = "well"
+  print(f"Hello, {x}, it works {y}!")
+  `);
+});
+
+test('non-null expression', () => {
+  expectPython([
```

Comment:
Maybe prepare for the next language: ```expect(input, { python: output })```

Reply:
Yep, totally planning to do that. It's the only way to cover all features across all languages.